apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -76,11 +82,12 @@ var TopInstructions = React.createClass({
     var id = this.props.id;
 
     var mainStyle = _.assign({}, styles.main, {
-      height: this.props.height - RESIZER_HEIGHT
-    });
-
-    var bodyStyle = _.assign({}, styles.body, {
-      height: mainStyle.height - styles.header.height,
+      height: this.props.height - RESIZER_HEIGHT,
+      // Visualization is hard-coded on embed levels. Do the same for instructions position
+      left: this.props.isEmbedView ? 340 : undefined
+    }, this.props.isEmbedView && {
+      height: undefined,
+      bottom: 0
```

Comment:
My eyes are glazing over at this.  Why do you have a ternary for `left` but rely on `_.assign` for `height` and `bottom`?  Couldn't this whole thing just be written:

```javascript
var mainStyle = _.assign({}, styles.main, {
  height: this.props.height - RESIZER_HEIGHT,
  left: undefined
}, this.props.isEmbedView && {
  height: undefined,
  bottom: 0,
  left: 340
});
```